src/components/License/License.tsx

```diff
@@ -18,71 +22,78 @@ type Props = {
   rights?: Rights[]
 }
 
+const cc2Icon = {
+  'cc': { icon: faCreativeCommons, alt: 'Creative Commons' },
+  'cc0': { icon: faCreativeCommonsZero, alt: 'Creative Commons Zero (Public Domain Dedication)' },
+  'by': { icon: faCreativeCommonsBy, alt: 'Creative Commons Attribution' },
+  'nc': { icon: faCreativeCommonsNc, alt: 'Creative Commons Non-Commercial' },
+  'nd': { icon: faCreativeCommonsNd, alt: 'Creative Commons No Derivatives' },
+  'sa': { icon: faCreativeCommonsSa, alt: 'Creative Commons Share Alike' }
+}
+const ccKeys = Object.keys(cc2Icon)
+
+const localContexts2Icon= {
+  'attribution-incomplete': { icon: lcCiAttribution, alt: 'Attribution Incomplete Notice' },
+  'bc-notice': { icon: lcBcNotice, alt: 'Biocultural (BC) Notice' },
+  'tk-notice': { icon: lcTkNotice, alt: 'Traditional Knowledge Notice' }
+}
+const localContextsKeys = Object.keys(localContexts2Icon)
+
+function isCreativeCommons( rights: Rights ) {
+  return rights.rightsIdentifier && rights.rightsIdentifier.startsWith('cc')
+}
+
+function isLocalContexts( rights: Rights ) {
+  return rights.rightsIdentifier && rights.rightsIdentifierScheme == "Local Contexts" && localContextsKeys.includes(rights.rightsIdentifier)
+}
+
 export const License: React.FunctionComponent<Props> = ({ rights = [] }) => {
 
   const uniqueRights = uniqBy([...rights], 'rightsIdentifier')
+
+  const localContextRigths = uniqueRights.reduce((sum, r) => {
```

Comment:
Not significant, but there's a typo here in `Rigths` if you want to rename this symbol.

Reply:
Good catch.  Done.